src/ontology/phenio.Makefile

```diff
@@ -17,24 +16,72 @@ RELEASE_ASSETS_AFTER_RELEASE=$(foreach n,$(RELEASE_ASSETS), ./$(n))
+
+
+################################################################
+#### Release files #############################################
+################################################################
+
 # Base file assembly
-$(TMPDIR)/$(ONT)-full-unreasoned.owl: $(SRC) $(OTHER_SRC)
+$(TMPDIR)/$(ONT)-full-unreasoned.owl: $(SRC) $(OTHER_SRC) $(IMPORT_FILES)
 	$(ROBOT) merge --input $< $(patsubst %, -i %, $(OTHER_SRC)) $(patsubst %, -i %, $(IMPORT_FILES)) \
-		--output $@
+		--annotate-derived-from true --output $@
 
 # Run a robot explain to check for unsatisfiable classes before next step
 $(EXPLAIN_OUT_PATH): $(TMPDIR)/$(ONT)-full-unreasoned.owl
 	$(ROBOT) explain -i $< -M unsatisfiability --unsatisfiable random:10 --explanation $@
 
-### Merge Biolink Model categories
-$(BLMODEL):
-	wget $(BLMODEL_URL) -O $@
-
-$(ONT)-full.owl: $(TMPDIR)/$(ONT)-full-unreasoned.owl | all_robot_plugins
+# removed this --update $(SPARQLDIR)/bl-categories.ru as we decided it was no longer needed here.
+phenio-full.owl: $(TMPDIR)/$(ONT)-full-unreasoned.owl $(MAPPINGDIR)/cl.sssom.tsv $(MAPPINGDIR)/uberon.sssom.tsv | all_robot_plugins
 	$(ROBOT) merge --input $< \
-		merge --input $(BLMODEL) \
-		query --update $(BLQUERY) \
-		unmerge --input $(BLMODEL) \
+		remove --select "<http://birdgenenames.org/cgnc/*>" \
+		remove --select "<http://flybase.org/reports/*>" \
+		remove --select "<http://purl.obolibrary.org/obo/CARO_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/ECTO_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/ENVO_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/FAO_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/GOCHE_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/MFOMD_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/MF_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/MOD_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/NCBITaxon_Union_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/NCIT_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/OBI_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/OGMS_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/PCO_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/PO_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/BTO_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/RnorDv_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/emapa#Tmp_new_group*>" \
+		remove --select "<http://www.ebi.ac.uk/efo/EFO_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/TS_*>" \
+		remove --select "<http://www.informatics.jax.org/marker/MGI:*>" \
+		remove --select "<https://swisslipids.org/rdf/SLM_*>" \
+		remove --select "<http://purl.obolibrary.org/obo/OPL_*>" \
+		remove --select "<https://bioregistry.io/lipidmaps*>" \
+		remove --term owl:Thing --term owl:Nothing \
+		rename --mappings config/property-map.tsv --allow-missing-entities true --allow-duplicates true \
```

Comment:
Not wanting to push too hard for my own software, but if you want to use a real SSSOM file for the “property map”, you could use the `sssom:rename` command of the SSSOM plugin.

That’s [what we do](https://github.com/obophenotype/uberon/blob/5b006f5af53983b4f2e03564a429c7e3e496accb/src/ontology/uberon.Makefile#L389) in Uberon, using [this mapping set](https://github.com/obophenotype/uberon/blob/master/src/mappings/import-corrections.sssom.tsv) as source.